### ISSUE TITLE
[6.19.2] stealable/purgable enemy buffs not working

### ISSUE THREAD

**Issue by Nukme** [Binbwen/tidyplates#6]
There are some lobster mages with water shield buffs (magic) on the shore near Seeker's Outpost in Zuldazar.

Just won't show whether this new feature is checked or not. TPC works fine here.

**Comment by Binbwen** [comment]
Did you enable via /hub > "Buffs & Debuffs" > "Show Purgable Buffs"?

It's not on by default.

**Comment by Nukme** [comment]
> Did you enable via /hub > "Buffs & Debuffs" > "Show Purgable Buffs"?
> 
> It's not on by default.

I don't see any option by that name. There's an option named "Include Stealable $ Purgable Enemey Buffs", and it's not working no matter on or off.

**Comment by Binbwen** [comment]
"Include Stealable.." is the one.  I forget the specific text, sometimes.  Regardless, I just tested it on the Bittertide Brinetail's, and Water Shield appears for me on my Shaman. 

I need a little more information, such as what class you are using, and if other buffs appear as expected.  Also, make sure you're configuring the right Hub profile.  I accidentally was changing settings on 'Tank' when I was using the 'Damage' profile.  Wondered why nothing was changing..

**Comment by Nukme** [comment]
Oh I tested it on my warrior. Is it possible that you set it to be seen only by classes that can actually purge these buffs?

I myself wish to be able to see these buffs as a tank, because sometimes I need to cry for help @-@.

**Comment by Luxocracy** [comment]
The purgeable attribute given to auras aren't properly set for a lot of magic buffs found in BFA, even though they can still be purged. So TPC simply just checks if the aura is magic or not, and assumes it can be purged, and I've yet to find any magic buff that can't be purged in BFA.

**Comment by Binbwen** [comment]
> Oh I tested it on my warrior. Is it possible that you set it to be seen only by classes that can actually purge these buffs?
> 
> I myself wish to be able to see these buffs as a tank, because sometimes I need to cry for help @-@.



> Oh I tested it on my warrior. Is it possible that you set it to be seen only by classes that can actually purge these buffs?
> 
> I myself wish to be able to see these buffs as a tank, because sometimes I need to cry for help @-@.

Yes - I'm using the Purgeable attribute to detect, which only shows up if you've got the ability to purge.

I see what you're saying about tanking, and wanting to call for assistance.
